deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -23,7 +23,13 @@ Java is used by the opensearch-migrations repo and Gradle, its associated build
 ```
 More details can be found [here](../../../TrafficCapture/dockerSolution/README.md)
 
-3- Fetch Migration Setup, in order to make use of Fetch Migration for historical data capture, a user should make any modifications necessary to the `dp_pipeline_template.yaml` file located in the same directory as this README before deploying. Further steps on starting Fetch Migration after deployment can be found [here](#kicking-off-fetch-migration)
+3- Fetch Migration Setup, in order to make use of Fetch Migration for historical data capture, a user should make any
+modifications necessary to the `dp_pipeline_template.yaml` file located in the same directory as this README before deploying.
+More information around the parameters used in the pipeline file can be found [here](https://opensearch.org/docs/latest/data-prepper/pipelines/pipelines/).
+
+The existing pipeline template works for the `demo-deploy` stack without any further modifications.
+
+Further steps on starting Fetch Migration after deployment can be found [here](#kicking-off-fetch-migration)
```

Comment:
nit: The new lines in this section seem a bit weird. In general we've taken the approach to keep things on a single line (even if a long line in the .md file) and let the viewer handle when to move text to the next line